### PR TITLE
CN-5042: amount in usd sent to mixpanel

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/GetCoinsEnterAmount/GetCoinsEnterAmount.js [hunk 1]
```diff
@@ -77,7 +78,16 @@ class GetCoinsEnterAmount extends Component {
...
+    const {
+      actions,
+      formData,
+      buyCoinsSettings,
+      currencyRatesShort,
+    } = this.props;
+
+    const cryptoProp = formData.cryptoCoin.toLowerCase();
+    const amountInUsd =
+      currencyRatesShort[cryptoProp] / Number(formData.amountCrypto);
```
Comment (fjovakaric) [inline app/components/screens/GetCoinsEnterAmount/GetCoinsEnterAmount.js:90]: it is `currencyRatesShort[cryptoProp] * Number(formData.amountCrypto)`

File: app/components/screens/GetCoinsEnterAmount/GetCoinsEnterAmount.js [hunk 2]
```diff
@@ -103,6 +113,16 @@ class GetCoinsEnterAmount extends Component {
         )} to continue.`
       );
     }
+
+    actions.updateFormField("amountInUsd", amountInUsd * 1.35); // NOTE: added 3.5% fee to value
```
Comment (fjovakaric) [inline app/components/screens/GetCoinsEnterAmount/GetCoinsEnterAmount.js:117]: it's `1.035`, but i'm not sure if we should add this
